hpx/util/tuple.hpp

```diff
@@ -848,6 +816,15 @@ namespace hpx { namespace util
         };
 
         ///////////////////////////////////////////////////////////////////////
+        /// A helper function for creating a non owning tuple from the given
+        /// arguments. Which means that the funcitonality of this function
+        /// lies between make_tuple and forward_as_tuple.
+        template <typename... T>
+        tuple<T...> create_raw_tuple(T&&... args)
+        {
+            return tuple<T...>{std::forward<T>(args)...};
```

Comment:
Use of list-init here makes me uneasy, but since no conversions ought to be involved it should be fine.